Added score viewing option and some style

devulapallisai opened the pull request:
I have added a score viewing option in the game and also added some styling like hiding scroll bar and all kind of stuff. If you like my work please give me hacktoberfest-accepted label and merge it

ANSHIKA1806 commented:
> I have added a score viewing option in the game and also added some styling like hiding scroll bar and all kind of stuff. If you like my work please give me hacktoberfest-accepted label and merge it

Please add screenshot of the changes that you made.

devulapallisai commented:
One sec

On Tue, Oct 5, 2021, 12:06 AM ANSHIKA ***@***.***> wrote:

> I have added a score viewing option in the game and also added some
> styling like hiding scroll bar and all kind of stuff. If you like my work
> please give me hacktoberfest-accepted label and merge it
>
> Please add screenshot of the changes that you made.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ANSHIKA1806/CardFlip_Game/pull/5#issuecomment-933749574>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUB4VV3LU4QFKS6ZVSQDAQTUFHXZ5ANCNFSM5FJ7FH3Q>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>

-- 


Disclaimer:- This footer text is to convey that this email is sent by one 
of the users of IITH. So, do not mark it as SPAM.

devulapallisai commented:
Added score one and hide the scrollbar

On Tue, Oct 5, 2021 at 12:18 AM DEVULAPALLI SAI PRACHODHAN <
***@***.***> wrote:

> One sec
>
> On Tue, Oct 5, 2021, 12:06 AM ANSHIKA ***@***.***> wrote:
>
>> I have added a score viewing option in the game and also added some
>> styling like hiding scroll bar and all kind of stuff. If you like my work
>> please give me hacktoberfest-accepted label and merge it
>>
>> Please add screenshot of the changes that you made.
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/ANSHIKA1806/CardFlip_Game/pull/5#issuecomment-933749574>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AUB4VV3LU4QFKS6ZVSQDAQTUFHXZ5ANCNFSM5FJ7FH3Q>
>> .
>> Triage notifications on the go with GitHub Mobile for iOS
>> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
>> or Android
>> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>>
>>
>

-- 


Disclaimer:- This footer text is to convey that this email is sent by one 
of the users of IITH. So, do not mark it as SPAM.

devulapallisai commented:
Hello madam, Could you please merge my pull request if it seems okay as soon as possible

devulapallisai commented:
![Memory Game - Google Chrome 10_5_2021 12_19_49 AM](https://user-images.githubusercontent.com/84134615/136019673-c9040edd-df09-4645-81df-692443f7c6d0.png)
I have hided scroll bar and added score option visible

ANSHIKA1806 commented:
@devulapallisai  
Hey, I have just accepted another pr so the original file is changed that's why there is a branch conflict with your pr ,
please fork again and copy paste your changes  and then make pr ,will accept your pr first
